core/src/components/select/select.ios.vars.scss

```
@@ -19,6 +19,9 @@ $select-ios-padding-start:          $item-ios-padding-start !default;
 /// @prop - Color of the select icon
 $select-ios-icon-color:             $text-color-step-650 !default;
 
+/// @prop - Size of the select icon
+$select-ios-icon-size: 18px !default;
```

```suggestion
$select-ios-icon-size:              18px !default;
```

Can we line up the variables, double check that my above suggestion looks aligned in the source file

---

We did away with this convention last year